vcr/issuer/openid_test.go

```diff
@@ -110,7 +110,7 @@ func Test_memoryIssuer_ProviderMetadata(t *testing.T) {
 
 	assert.Equal(t, openid4vci.ProviderMetadata{
 		Issuer:        "https://example.com/did:nuts:issuer",
-		TokenEndpoint: "https://example.com/did:nuts:issuer/oidc/token",
+		TokenEndpoint: "https://example.com/did:nuts:issuer/token",
```

Comment:
How come the tests do not have `/n2n/identity` in it?

Reply:
Since we didn't include it in the issuer identifier in the test (line 42)